include/exadg/structure/spatial_discretization/operators/linear_operator.cpp

```diff
@@ -35,11 +35,14 @@ LinearOperator<dim, Number>::do_cell_integral(IntegratorCell & integrator) const
 
   for(unsigned int q = 0; q < integrator.n_q_points; ++q)
   {
-    // engineering strains (material tensor is symmetric)
-    tensor const gradient = integrator.get_gradient(q);
+    // engineering strains
+    tensor const symmetric_gradient = integrator.get_symmetric_gradient(q);
 
-    // Cauchy stresses
-    tensor const sigma = material->apply_C(gradient, integrator.get_current_cell_index(), q);
+    // Cauchy stresses, only valid for linear elasticity
+    tensor const sigma =
```

Comment:
also note here that I am not using the `second_piola_kirchhoff_stress_derivative` here to have a single implementation for linear elasticity and finite strain elasticity using the St Venant Kirchhoff model.